thirdparty/build-thirdparty.sh

```diff
@@ -1412,6 +1423,7 @@ build_fiu
 build_llvm
 build_clucene
 build_simdutf
+build_tenann
```

Comment:
make this step conditional. only do the build for x86 platform. check line 1428~1431

Reply:
Got it